backend/persistent/datastores/Datastores.js

```diff
@@ -70,13 +71,24 @@ export function query(queryObject, connection) {
 }
 
 /*
- * connect functions attempt to ping the connection and
- * return a promise that is empty
+ * connect attempts to ping the connection and
+ * returns a promise that resolves to the connection object
```

Comment:
@returns {promise} an empty promise

Reply:
This depends on the connector and it's likely to change once Falcon becomes smarter about the connections it creates <sup>(at the moment, most of the connectors create a new connection for every request, which is OK if the DB driver implements a pool or the request is stateless)</sup>.

I'll reword it, so that it only states that the promise resolves once the connection succeeds.